llm_guard/resources/sensisitive_patterns.json

```diff
@@ -47,6 +51,41 @@
       "987-65-4321"
     ],
     "context": [],
-    "score": 0.75
+    "score": 0.75,
+    "languages": ["en"]
+  },
+
+  {
+    "name": "CREDIT_CARD",
+    "reuse": {"language": "en", "name": "CREDIT_CARD"},
```

Comment:
Is it to reuse the default Presidio pattern? Could we do the same for Email and IP?

Reply:
I have tried, the patterns of Email and IP not working for Chinese, because there are used `\b` `\W` regex, but Chinese characters are Unicode in regex. 

The CREDIT_CARD all numbers and `-`, so it can be reused.